modules/dfpAdServerVideo.js

```diff
@@ -55,9 +56,26 @@ const defaultParamConstants = {
  *   demand in DFP.
  */
 export default function buildDfpVideoUrl(options) {
+  if (!options.params && !options.url) {
```

Comment:
Looks like the JSDoc types need some updating for this. The `DfpVideoOptions` still says that `params` is required, and doesn't mention `url` at all.